cmd/conformance/mysql/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.22.7-bookworm@sha256:027bd04b1d3b4529bf8ccebf62eb7eeeae7b7bef134a68bd419824e929ad93ad AS build
+FROM golang:1.23.4-bookworm@sha256:027bd04b1d3b4529bf8ccebf62eb7eeeae7b7bef134a68bd419824e929ad93ad AS build
```

Comment:
The hash should be changed when the tag is changed.

Reply:
whoops, ta!